package.json

```diff
@@ -14,13 +14,21 @@
     "next": "13.4.19",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "typescript": "5.2.2",
-    "sitefinity-react-framework": "file:./framework/build/sitefinity-react-framework-0.1.0.tgz"
+    "sitefinity-react-framework": "file:framework/build/sitefinity-react-framework-0.1.0.tgz",
+    "typescript": "5.2.2"
   },
   "scripts": {
     "prod": "next start",
     "dev": "node server.js",
     "build": "next build",
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest --ci"
+  },
+  "devDependencies": {
+    "@testing-library/jest-dom": "^6.1.4",
```

Comment:
We usually pin the versions of all our dependencies, please consider it here as well.

Reply:
Fixed.